### PR TITLE
SQFT microservice

### PR DESCRIPTION
**Pull request by jpablomch**
Enables SQFT microservices for external frameworks.

### CODE REVIEW

File: SQFT/install.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ pip install git+https://github.com/AutoGPTQ/AutoGPTQ@866b4c8
 
 # nncf
 git clone https://github.com/openvinotoolkit/nncf.git
-cd nncf && git checkout v2.12.0 && git apply --ignore-space-change --ignore-whitespace ${SQFT_PATH}/patches/nncf-v2.12.0.patch && pip install -e . && cd ..
+cd nncf && git checkout f143e1c && git apply --ignore-space-change --ignore-whitespace ${SQFT_PATH}/patches/nncf-f143e1c.patch && pip install -e . && cd ..
```
Comment (michaelbeale-IL) [inline SQFT/install.sh:24]: Do you have to be tied to a commit and not a full release?

File: SQFT/install.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ pip install git+https://github.com/AutoGPTQ/AutoGPTQ@866b4c8
 
 # nncf
 git clone https://github.com/openvinotoolkit/nncf.git
```
Comment (michaelbeale-IL) [inline SQFT/install.sh:23]: Is this listed in our BOM?

**Review by michaelbeale-IL — COMMENTED** [review]
General comment:
It doesn't look like you're redistributing any new datasets, but are they in our AI Inventory somewhere already?

File: SQFT/install.sh (hunk 1 above)
Comment (jpablomch) [reply SQFT/install.sh:24]: Yes, we are using latest features in the development branch. Thanks!

**Comment by jpablomch** [comment]
> General comment: It doesn't look like you're redistributing any new datasets, but are they in our AI Inventory somewhere already?

Nothing new in our AI inventory. We are using the same datasets that we originally used with SQFT. Thanks!

**Review by michaelbeale-IL — APPROVED** [review]
LGTM

**Pull request merged**